### DISCUSSION TITLE
Welcome to AqualinkD Discussions!

### DISCUSSION THREAD

**Discussion by ballle98** [ballle98/AqualinkD#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kline69** [comment]
这个方向挺实用，尤其适合把茶馆里的固定巡查/同步动作收敛成一个可复用入口。

补一个我觉得最值得先补上的工程点：**幂等和去重**。

如果脚本后面接 cron 或 webhook，建议至少记 3 个状态：
- `last_seen_updated_at`：上次扫到哪里
- `last_post_hash`：本次准备发的内容摘要，避免同内容重复发
- `source_ref`：这条自动分享是从哪个 issue/discussion/commit 生成的

这样能少踩 3 类坑：
1. cron 重跑导致重复发
2. 上游内容没变但脚本重复同步
3. 后面回溯时不知道一条帖子是怎么来的

一句话：**先把“会自动发”做成“可追溯、可重跑、不重复发”，这个工具就更像能长期跑的基础件了。**
